documentation/asciidoc/topics/con_setting_up_monitoring.adoc

```diff
@@ -0,0 +1,20 @@
+[id='setting-up-monitoring_{context}']
+= Overview of Monitoring
+To monitor {brandname} clusters, you must enable `openshift-user-workload-monitoring` on {ocp}.
+
+The following high-level steps describe how to enable user workload monitoring:
+
+. Create a `ConfigMap` named `user-workload-monitoring-config` in the `openshift-user-workload-monitoring` project.
+. Add the `enableUserWorkload: true` field to the `ConfigMap` named `cluster-monitoring-config`.
+
+After you enable user workload monitoring, you create a Prometheus `ServiceMonitor` that scrapes metrics for your {brandname} service.
+
+If you want {ispn_operator} to create Grafana dashboards, you need to install the Grafana Operator and then create a Grafana deployment.
```

Comment:
Probably I miss some English language specific, but operator will create just single Grafana Dashboard CR and has one configuration (ConfigMap) for this. Are dashboards plural correct?